src/cktaba.f

```diff
@@ -8,14 +8,14 @@
 C> from Section 1 of a BUFR message that was previously read from LUN
 C> using one of the [message-reading subroutines](@ref hierarchy).
 C>
-C> @param[in] LUN - integer: file ID.
+C> @param[in] LUN - integer: file ID
 C> @param[out] SUBSET - character*8: Table A mnemonic
 C>                      - returned as a string of all blank characters
 C>                        if IRET is equal to 11 (see below) and if Section 3
 C>                        isn't being used for decoding
 C> @param[out] JDATE    - integer: date-time stored within Section 1 of BUFR
 C>                        in format of either YYMMDDHH or
-C>                        YYYYMMDDHH, depending on datelen() value.
+C>                        YYYYMMDDHH, depending on datelen() value
```

Comment:
How come you are taking away the punctuation?

Reply:
B/C some time ago you yourself had pointed out that these variable descriptions usually aren't complete sentences and therefore don't really need periods after them :-)

Reply:
OK, just curious.